GECKOInstaller.m

```diff
@@ -1,14 +1,31 @@
 classdef GECKOInstaller
-% GECKOInstaller
-%   Support for installing and uninstalling
-%   Run GECKOInstaller.install to install (will set up the path in MATLAB)
-%   Run GECKOInstaller.uninstall to clear the path from MATLAB
-%   To install, you first need to cd to the GECKO root.
+    % GECKOInstaller
```

Comment:
Running `GECKOInstaller.install` I got:

Error using dir
Pathname
'/Users/chenyu/Documents/GitHub/GECKO:/Users/chenyu/Documents/GitHub/GECKO/Databases:/Users/chenyu/Documents/GitHub/GECKO/doc:/Users/chenyu/Documents/GitHub/GECKO/doc/src:/Users/chenyu/Documents/GitHub/GECKO/doc/src/geckomat:/Users/chenyu/Documents/GitHub/GECKO/doc/src/geckomat/change_model:/Users/chenyu/Documents/GitHub/GECKO/doc/src/geckomat/gather_kcats:/Users/chenyu/Documents/GitHub/GECKO/doc/src/geckomat/get_enzyme_data:/Users/chenyu/Documents/GitHub/GECKO/doc/src/geckomat/kcat_sensitivity_analysis:/Users/chenyu/Documents/GitHub/GECKO/doc/src/geckomat/kcat_sensitivity_analysis/Bayesian:/Users/chenyu/Documents/GitHub/GECKO/doc/src/geckomat/limit_proteins:/Users/chenyu/Documents/GitHub/GECKO/doc/src/geckomat/model_adapter:/Users/chenyu/Documents/GitHub/GECKO/doc/src/geckomat/utilities:/Users/chenyu/Documents/GitHub/GECKO/doc/src/geckomat/utilities/ecFSEOF:/Users/chenyu/Documents/GitHub/GECKO/src:/Users/chenyu/Documents/GitHub/GECKO/src/dlkcat-gecko:/Users/chenyu/Documents/GitHub/GECKO/src/dlkcat-gecko/input:/Users/chenyu/Documents/GitHub/GECKO/src/geckomat:/Users/chenyu/Documents/GitHub/GECKO/src/geckomat/change_model:/Users/chenyu/Documents/GitHub/GECKO/src/geckomat/gather_kcats:/Users/chenyu/Documents/GitHub/GECKO/src/geckomat/get_enzyme_data:/Users/chenyu/Documents/GitHub/GECKO/src/geckomat/kcat_sensitivity_analysis:/Users/chenyu/Documents/GitHub/GECKO/src/geckomat/kcat_sensitivity_analysis/Bayesian:/Users/chenyu/Documents/GitHub/GECKO/src/geckomat/limit_proteins:/Users/chenyu/Documents/GitHub/GECKO/src/geckomat/model_adapter:/Users/chenyu/Documents/GitHub/GECKO/src/geckomat/utilities:/Users/chenyu/Documents/GitHub/GECKO/src/geckomat/utilities/ecFSEOF:/Users/chenyu/Documents/GitHub/GECKO/src/geckopy:/Users/chenyu/Documents/GitHub/GECKO/src/geckopy/brenda_parser:/Users/chenyu/Documents/GitHub/GECKO/test:/Users/chenyu/Documents/GitHub/GECKO/test/manual_tests:/Users/chenyu/Documents/GitHub/GECKO/test/unit_tests:/Users/chenyu/Documents/GitHub/GECKO/test/unit_tests/ecTestGEM:/Users/chenyu/Documents/GitHub/GECKO/test/unit_tests/ecTestGEM/data:/Users/chenyu/Documents/GitHub/GECKO/test/unit_tests/ecTestGEM/models:/Users/chenyu/Documents/GitHub/GECKO/userData:/Users/chenyu/Documents/GitHub/GECKO/userData/ecYeastGEM:/Users/chenyu/Documents/GitHub/GECKO/userData/yourModel:/Users/chenyu/Documents/GitHub/GECKO/userData/yourModel/data:/Users/chenyu/Documents/GitHub/GECKO/userData/yourModel/models/*.m'
is too long.

Error in checkFunctionUniqueness (line 29)
        temp_res=dir(fullfile(char(altDirs(i)),'*.m'));

Error in GECKOInstaller.install (line 22)
                status=checkFunctionUniqueness(paths);

Reply:
As a test, could you replace in `checkFunctionUniqueness`, line 26 from:

```matlab
    altDirs = strsplit(altDirs,';')';
```

to

```matlab
    if ispc
        splitChar = ';';
    else
        splitChar = ':';
    end
    altDirs = strsplit(altDirs,splitChar)';
```

Reply:
Now it works!

Reply:
There is an approved pull request for this in RAVEN https://github.com/SysBioChalmers/RAVEN/pull/491, so I believe it's safe to resolve this comment (please reopen if otherwise).

Reply:
Ok, RAVEN `devel` branch is then required for the moment, will make a 2.8.3 release when making the next GECKO release.